### ISSUE TITLE
Where to get RakNet Support ?

### ISSUE THREAD

**Issue by meiry** [OculusVR/RakNet#100]
As the forum looks dead , and massages here are barely  get answered , 
Where are the community of RakNet are Usually? 
Tישמלד

**Comment by Mellnik** [comment]
All the community I know is this Github repository. The forums are flooded with porn spam since Oculus bought RakNet in 2014. Ever since no progress was made.
Someone who is interested in maintaining this project should get push permission.

**Comment by meiry** [comment]
Yeah i noticed ... if no support so maybe its time to move to other solution , Although this lib have so much in it .
any recommendation for free ios/android/desktop udp client ?

**Comment by Kiddinglife** [comment]
go to see this branch lots of updates
https://github.com/larku/RakNet
